src/main/java/racing/domain/ParticipateCars.java

```diff
@@ -0,0 +1,78 @@
+package racing.domain;
+
+import racing.util.RandomGenerator;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class ParticipateCars {
+    private static final List<Car> cars = new ArrayList<>();
+
+    public ParticipateCars() {
+    }
+
+    public ParticipateCars(String[] cars) {
+        createCar(cars);
+    }
+
+    protected List<Car> createCar(String[] carNames) {
+        for (String name : carNames) {
+            this.cars.add(new Car(name));
+        }
+        return new ArrayList<>(this.cars);
+    }
+
+    public void moveCar() {
+        for (Car car : this.cars) {
+            int randomNumber = RandomGenerator.getRandomNumber();
+            car.run(randomNumber);
+        }
+    }
+
+    public List<Car> getCars() {
+        return new ArrayList<>(this.cars);
+    }
+
+    public String winner() {
+        int numOfWinnerByCondition = 2;
+        int maxTotalMeter = findMaxTotalMeter(cars);
+        List<String> winner = findWinnerName(maxTotalMeter,cars);
+        return winnerName(winner, numOfWinnerByCondition);
+    }
+
+    protected List<String> findWinnerName(int maxTotalMeter,List<Car> cars) {
+        List<String> winner = new ArrayList<>();
+        cars.forEach( (car)->{
+            if(car.totalMeter() == maxTotalMeter){
+                winner.add(car.getName());
+            }
+        });
+        return winner;
+    }
+
+    protected int findMaxTotalMeter(List<Car> cars){
+        Comparator<Car> comparable = (carA, carB) -> carA.compareMeter(carB);
+        Optional<Car> carOptional = cars.stream().max(comparable);
+        if(carOptional.isPresent()){
+            return carOptional.get().totalMeter();
+        }
+        throw new RuntimeException("자동차가 존재하지 않습니다.");
```

Comment:
Optinal을 사용하지 않고 다음처럼 값을 구해볼 수 있을 것 같아요!
```suggestion
        return cars.stream()
                   .max(Car::compareMeter)
                   .map(Car::totalMeter)
                   .get();
```